### PR TITLE
Added initial streams rest API

### PR DESCRIPTION
**Pull request by wmarshall484**

### CODE REVIEW

**Comment by ddebrunner** [comment]
Can you remove the pyc cached items?

**Comment by ddebrunner** [comment]
Just wondering if streamsx should be the top-level, or there should be a python first?

**Comment by ddebrunner** [comment]
Maybe `python/packages/streamsx` to have a clear indication of where the actual code is and have items like `python/samples` in the future.

**Comment by ddebrunner** [comment]
Any plan to address the comments?

**Comment by wmarshall484** [comment]
Removed the cached python files, added a samples directory, and added a sample outlining basic usage.

File: python/packages/streamsx/rest.py [hunk 1]
```diff
@@ -0,0 +1,537 @@
+import requests
+from pprint import pprint, pformat
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+
+class StreamsRestConnection(object):
+    """
```
Comment (ddebrunner) [inline python/packages/streamsx/rest.py]: Is the comment in the wrong place, StreamsRestClient, not connectioin?

**Comment by wmarshall484** [comment]
TODO: make certain classes private.
            remove 'streams' from 'streams_rest_utils'
            Fix/add documentation

**Pull request merged**